[2021-12-25 11:19:37,563] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:19:33.453920+00:00 [queued]>
[2021-12-25 11:19:37,599] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:19:33.453920+00:00 [queued]>
[2021-12-25 11:19:37,600] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:19:37,600] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-12-25 11:19:37,601] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:19:37,634] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): myfunction> on 2021-12-25T11:19:33.453920+00:00
[2021-12-25 11:19:37,643] {standard_task_runner.py:52} INFO - Started process 47 to run task
[2021-12-25 11:19:37,658] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'survey', 'myfunction', '2021-12-25T11:19:33.453920+00:00', '--job-id', '183', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/survey.py', '--cfg-path', '/tmp/tmpi3gjfa_9', '--error-file', '/tmp/tmpk5tuds19']
[2021-12-25 11:19:37,659] {standard_task_runner.py:77} INFO - Job 183: Subtask myfunction
[2021-12-25 11:19:37,831] {logging_mixin.py:104} INFO - Running <TaskInstance: survey.myfunction 2021-12-25T11:19:33.453920+00:00 [running]> on host e1668ec19f6d
[2021-12-25 11:19:37,954] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=survey
AIRFLOW_CTX_TASK_ID=myfunction
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T11:19:33.453920+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-25T11:19:33.453920+00:00
[2021-12-25 11:19:37,961] {python.py:118} INFO - Done. Returned value was: None
[2021-12-25 11:19:38,008] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=survey, task_id=myfunction, execution_date=20211225T111933, start_date=20211225T111937, end_date=20211225T111938
[2021-12-25 11:19:38,103] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-25 11:19:38,147] {local_task_job.py:146} INFO - Task exited with return code 0
